<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1574</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="homeManagerScreen_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1575</width>
     <height>900</height>
    </rect>
   </property>
   <widget class="QWidget" name="homeTopBar_widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>20</y>
      <width>1251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
border-radius:20px;</string>
    </property>
    <widget class="QLabel" name="homeTopBarTitle_label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>18</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins SemiBold</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(64, 64, 64);</string>
     </property>
     <property name="text">
      <string>Movie Recommender</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>10</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#exit_btn{
background-color: rgb(229, 108, 120);
	color: rgb(230, 230, 230);
border-radius:20px;
}

QPushButton#exit_btn:hover{
background-color: rgba(229, 108, 120,220);
}

QPushButton#exit_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(229, 108, 120,220);
}</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="homeContainer_widget" native="true">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>120</y>
      <width>1251</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
border-radius:20px;</string>
    </property>
    <widget class="QWidget" name="DashboardContainer_widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>192</y>
       <width>1241</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,230);
border-radius:20px;</string>
     </property>
     <widget class="QPushButton" name="recommend_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>1181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins Medium</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#recommend_btn{
	background-color: rgba(237, 172, 72,40);
	
	
	color: rgb(237, 172, 72);
border-radius:20px;
}

QPushButton#recommend_btn:hover{
background-color: rgba(237, 172, 72,60);

}

QPushButton#recommend_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(237, 172, 72,60);
}</string>
      </property>
      <property name="text">
       <string> Recommend Movies</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/materials/icons/natural-language-processing.png</normaloff>:/icons/materials/icons/natural-language-processing.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QListWidget" name="recomendedMovies_listWidget">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>91</y>
        <width>1171</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="top10_title_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins Medium</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(64, 64, 64);</string>
      </property>
      <property name="text">
       <string>Top 20 Trending Movies :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DashboardContainer_widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>1241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,230);
border-radius:20px;</string>
     </property>
     <widget class="QLabel" name="home_label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins Medium</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(64, 64, 64);</string>
      </property>
      <property name="text">
       <string>Enter your favorite movie name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="movieTitle_tbox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>511</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgb(229,108,120);
color:rgpa(255,255,255,230);
padding-bottom:7px;
border-radius:0px;


</string>
      </property>
      <property name="placeholderText">
       <string>Movie title</string>
      </property>
     </widget>
     <widget class="QPushButton" name="randomMovieName_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins Medium</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton#randomMovieName_btn{
	background-color: rgba(108, 127, 254,30);
	color: rgb(108, 127, 254);
border-radius:20px;
}

QPushButton#randomMovieName_btn:hover{
background-color: rgba(108, 127, 254,50);
color: rgb(108, 127, 254);
}

QPushButton#randomMovieName_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(108, 127, 254,50);
color: rgb(108, 127, 254);
}</string>
      </property>
      <property name="text">
       <string>Mock test</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="mainScreenwhite_BG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1575</width>
      <height>900</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 242, 243);
border-radius:38px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="homeSideBar_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>901</height>
     </rect>
    </property>
    <widget class="QLabel" name="homeSideBar_BG">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>901</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/materials/homeSideMenu.png);
border-top-left-radius:38px;
border-bottom-left-radius:38px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>700</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#search_btn{
	
	background-color: rgba(237, 239, 254,35);
	color: rgb(255,255,255);
border-radius:20px;
}

QPushButton#search_btn:hover{
background-color: rgba(237, 239, 254,100);
}

QPushButton#search_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(237, 239, 254,100);
}</string>
     </property>
     <property name="text">
      <string> Search by Catergory</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/materials/icons/search.png</normaloff>:/icons/materials/icons/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="homeAboutUs_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>790</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#homeAboutUs_btn{
	
	
	background-color: rgba(41, 94, 154,150);
	color: rgb(255,255,255);
border-radius:20px;
}

QPushButton#homeAboutUs_btn:hover{
background-color: rgba(237, 239, 254,100);
}

QPushButton#homeAboutUs_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(237, 239, 254,100);
}</string>
     </property>
     <property name="text">
      <string> About Us    </string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/materials/icons/info7.png</normaloff>:/icons/materials/icons/info7.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="homeUserImage_Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>309</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
border-image: url(:/icons/materials/icons/radio-waves.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="homeUserName_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Enter a movies genre:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="movieCategory_tbox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgb(255,255,255);
color:rgpa(255,255,255,230);
padding-bottom:7px;
border-radius:0px;



</string>
     </property>
     <property name="placeholderText">
      <string>Movie category</string>
     </property>
    </widget>
   </widget>
   <zorder>mainScreenwhite_BG</zorder>
   <zorder>homeTopBar_widget_2</zorder>
   <zorder>homeContainer_widget</zorder>
   <zorder>homeSideBar_widget</zorder>
  </widget>
  <widget class="QWidget" name="dataEntry_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1575</width>
     <height>900</height>
    </rect>
   </property>
   <widget class="QLabel" name="mainScreen_BG_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1575</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/materials/dataEntryBG.png);
border-radius:38px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="topBar_widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>1351</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(200, 200, 200,80);
/*background-color: rgba(239, 174, 74,150);*/
border-radius:20px;</string>
    </property>
    <widget class="QLabel" name="mainTopBarTitle_label_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins SemiBold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(235, 168, 105);</string>
     </property>
     <property name="text">
      <string>Dashboard</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dataEntryBack_btn">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>10</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#dataEntryBack_btn{
background-color: rgb(229, 108, 120);
	color: rgb(230, 230, 230);
border-radius:20px;
}

QPushButton#dataEntryBack_btn:hover{
background-color: rgba(229, 108, 120,220);
}

QPushButton#dataEntryBack_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(229, 108, 120,220);
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainDashboardContainer_widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>1531</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(200, 200, 200,80);
border-radius:20px;</string>
    </property>
    <widget class="QPushButton" name="dataEntrySave_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>660</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#dataEntrySave_btn{
	
	background-color: rgb(235, 168, 105);
	color: rgb(255, 255, 255);
border-radius:20px;
}

QPushButton#dataEntrySave_btn:hover{
background-color: rgba(245, 178, 115,210);

}

QPushButton#dataEntrySave_btn:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(245, 178, 115,210);

}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="mainTopBarTitle_label_3">
     <property name="geometry">
      <rect>
       <x>1420</x>
       <y>660</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(64, 64, 64);</string>
     </property>
     <property name="text">
      <string>ScMS</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="mainScreenDark_BG_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1575</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,5);
border-radius:38px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>mainScreen_BG_2</zorder>
   <zorder>mainScreenDark_BG_2</zorder>
   <zorder>topBar_widget_2</zorder>
   <zorder>mainDashboardContainer_widget_2</zorder>
  </widget>
  <zorder>dataEntry_widget</zorder>
  <zorder>homeManagerScreen_widget</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
